src/fpe/fpe.ts

```diff
@@ -0,0 +1,163 @@
+import init, {
+  webassembly_fpe_decrypt_alphabet,
+  webassembly_fpe_decrypt_big_integer,
+  webassembly_fpe_decrypt_float,
+  webassembly_fpe_decrypt_integer,
+  webassembly_fpe_encrypt_alphabet,
+  webassembly_fpe_encrypt_big_integer,
+  webassembly_fpe_encrypt_float,
+  webassembly_fpe_encrypt_integer,
+} from "../pkg/fpe/cloudproof_fpe"
+
+let initialized: Promise<void> | undefined
+
+let wasmInit: (() => any) | undefined
+export const setFpeInit = (arg: () => any): void => {
+  wasmInit = arg
+}
+
+// Use an interface to define the `encrypt` and `decrypt` options
+interface FpeOptions {
+  alphabet?: string
+  additionalCharacters?: string
+  radix?: number
+  digits?: number
+}
+
+// Use an enum to define the valid alphabet IDs
+enum FpeAlphabetId {
```

Comment:
not consistent with Rust side of alphabet listing and naming :)

Reply:
right